spk/erlang/Makefile

```
@@ -14,6 +14,8 @@ DISPLAY_NAME = Erlang
 HOMEPAGE   = http://www.erlang.org
 LICENSE    = MPL
 
+SPK_LINKS = /usr/local/bin/erl:bin/erl /usr/local/bin/erlc:bin/erlc
```

please prefer `SPK_COMMANDS` over `SPK_LINKS` (i.e. SPK_LINKS is required only for different source and target names) 
```suggestion
SPK_COMMANDS = bin/erl bin/erlc
```

`SPK_LINKS` will not be supported with DSM 7+